### ISSUE TITLE
plot history? [Question] 

### ISSUE THREAD

**Issue by cmeaton** [BrikerMan/Kashgari#322]
Thank you for the wonderful software, its really great. How do I plot the test/val accuracy and loss history at every step? I know there is functionality with tensorboard, but I want to visualize with matplotlib. With keras, I use this code to visualize:

`def plot_metrics(history):

    # summarize history for accuracy
    plt.plot(history.history['acc'])
    plt.plot(history.history['val_acc'])
    plt.title('model accuracy')
    plt.ylabel('acc')
    plt.xlabel('epoch')
    plt.legend(['train', 'test'], loc='upper left')
    plt.show()
    # summarize history for loss
    plt.plot(history.history['loss'])
    plt.plot(history.history['val_loss'])
    plt.title('model loss')
    plt.ylabel('loss')
    plt.xlabel('epoch')
    plt.legend(['train', 'test'], loc='upper left')
    plt.show()`

Is it possible to do this with a Kashgari model?

**Comment by BrikerMan** [comment]
Yep, you can. Just do this.

```
history = model.fit(x, y, ....)
# Then you can plot it.
# But it might be `history.history['accuracy']` not `history.history['acc']`
```
